addons/mail/static/src/messaging/component/chatter/chatter.xml

```diff
@@ -40,6 +40,7 @@
                             hasComposer="false"
                             haveMessagesAuthorRedirect="true"
                             order="'desc'"
+                            hasScrollAdjust="!chatter.hasExternalBorder"
```

Comment:
We need to consider renaming `hasExternalBorder` or creating a new prop `hasScrollAdjust` because it's not good to use the "border" named-prop to control "scroll".